src/elements/Tag/Tag.spec.tsx

```diff
@@ -0,0 +1,59 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+import React from "react";
+import { mount } from "@cypress/react";
+import { Tag, TagType } from "./Tag";
+
+const TAG_ID = "[data-test-id=tag]";
+const TAG_LABEL = "My awesome tag";
+const TAG_REJECT_ICON_ATTRIBUTE = "svg[name=IconReject]";
+
+describe("Tag Component", () => {
+    it("should render a label", () => {
+        mount(<Tag type={TagType.Suggested} label={TAG_LABEL} />);
+
+        cy.get(TAG_ID).contains(TAG_LABEL);
+    });
+    it("should render a suggested tag", () => {
```

Comment:
Maybe a bit of spacing for better readability?

```suggestion
    });

    it("should render a suggested tag", () => {
```